lib/qs/job.rb

```diff
@@ -32,10 +33,9 @@ def inspect
 
     def ==(other)
       if other.kind_of?(self.class)
-        self.payload_type == other.payload_type &&
```

Comment:
@kellyredding - This is jumping the gun a little bit but once event no longer composes job this isn't needed. The `kind_of` check will cover checking for the payload type because all `Job` payload type will be the same.

I can restore it for now.